lib/src/create_report.dart

```diff
@@ -71,6 +79,113 @@ environment:
           basePath: packageDir));
 }
 
+Future<ReportSection> _staticAnalysis(
+  String packageDir,
+  ToolEnvironment toolEnvironment, {
+  @required bool usesFlutter,
+}) async {
+  final analysisResult = await _analyzePackage(packageDir, toolEnvironment,
+      usesFlutter: usesFlutter);
+
+  final errors = analysisResult.errors;
+  final warnings = analysisResult.warnings;
+  final lints = analysisResult.lints;
+
+  // Only try to run dartfmt if there where no errors.
+  final formattingIssues = errors.isEmpty
+      ? await _formatPackage(packageDir, toolEnvironment,
+          usesFlutter: usesFlutter)
+      : <_Issue>[];
+
+  return ReportSection(
+    title: 'Code follows recommended code style',
+    maxPoints: 20,
+    grantedPoints: (errors.isEmpty && warnings.isEmpty)
+        ? (formattingIssues.isEmpty && lints.isEmpty ? 20 : 10)
+        : 0,
+    summary: _makeSummary('''
+10 points: code has no errors and warnings.
```

Comment:
Nit: extra line break and `*10 points:*` (`<em>` tag) + same for `20 points`

Reply:
Done